src/app/frontend/replicasetdetail/replicasetdetail_controller.js

```diff
@@ -156,6 +165,19 @@ export default class ReplicaSetDetailController {
    */
 
   /**
+   * Updates replicas count in replica set
+   * @param {number} replicasCount
+   * @param {function(!backendApi.ReplicaSetDetail)=} opt_callback
+   * @param {function(!angular.$http.Response)=} opt_errback
+   * @private
+   */
+  updateReplicas_(replicasCount, opt_callback, opt_errback) {
+    this.replicaSetDetail.podsDesired = replicasCount;
```

Comment:
You think it is a good idea to actually send `backendApi.ReplicaSetDetail` but only with update property filled? Endpoint could be `/replicasets/:namespace/:name/update/pods`. Last part can be changed to what is actually updated, so `pods`, `name` etc.